book/09-git-and-other-scms/sections/client-bzr.asc

```diff
@@ -74,11 +74,28 @@ $ git fetch
 
 ===== Ignore what is ignored with .bzrignore
 
-As the format of the `.bzrignore` file is completely compatible with `.gitignore`'s one, and as you shouldn't make a `.gitignore` file in your repository, it is enough to make a symbolic link to `.bzrignore` so that the potential changes of `.bzrignore` are taken into account:
-[source,console]
-----
-$ ln -s .bzrignore .git/info/exclude
-----
+Since you are working on a project managed with Bazaar, you shouldn't create a `.gitignore` file because you _may_ accidentally set it under version control and the other people working with Bazaar would be disturbed.
+The solution is to create the `.git/info/exclude` file either as a symbolic link or as a regular file.
+We'll see later on how to solve this question.
+
+Bazaar uses the same model as Git to ignore files, but also has two features which don't have an equivalent into Git.
+The complete description may be found in http://doc.bazaar.canonical.com/bzr.2.7/en/user-reference/ignore-help.html[the documentation].
+The two features are:
+
+1. "!!" at the beginning of a character string takes prevail on "!" at the beginning of the string, which allows to ignore some files that would have been included with "!"
+2. character strings starting with "RE:".
+What follows "RE:" is a http://doc.bazaar.canonical.com/bzr.2.7/en/user-reference/patterns-help.html[regular expression].
+Git does not allow the use of regular expressions, only shell globs.
```

Comment:
Should I let the link to the Bazaar patterns which specifies that the regular expressions are Python regular expressions ?

Reply:
If you can find a way to make it fit in the sentence, but it’s not too important I think.

Reply:
Maybe I could make a link to the Python regex library instead of a link to Bazaar patterns ?
What do you think about this:
> "RE:" at the beginning of a line allows you to specify a [Python regular expression](https://docs.python.org/3/library/re.html) (Git only allows shell globs)

?

Reply:
😍